### ISSUE TITLE
Learning rate and optimizer used for the VQ-VAE-2 paper

### ISSUE THREAD

**Issue by netw0rkf10w** [deepmind/sonnet#255]
Hello,

I would like to know if you could share the details of the optimization hyper-parameters used in the VQ-VAE-2 paper, including the optimizer and learning rate (value and schedule). The details given in the paper do not include this information:

<img width="880" alt="Screen Shot 2022-10-26 at 13 31 53" src="https://user-images.githubusercontent.com/8569515/198015545-07e89fba-a9b0-4052-b815-a021e8f8ad92.png">

Thank you in advance for your response.

**Comment by tomhennigan** [comment]
Hi, the authors are not core contributors to Sonnet so don't read the issues posted here. I would suggest reaching out to them directly, their email addresses are at the top of the paper.

**Comment by netw0rkf10w** [comment]
Thanks, @tomhennigan. I'm going to reach out to the authors directly.
